plugins/com.google.cloud.tools.eclipse.appengine.validation/src/com/google/cloud/tools/eclipse/appengine/validation/XmlSourceValidator.java

```diff
@@ -118,7 +118,7 @@ public void setInitializationData(IConfigurationElement config, String propertyN
       String className = (String) data;
       Class<?> clazz = Class.forName(className);
       Constructor<?> constructor = clazz.getConstructor();
-      this.setHelper((XmlValidationHelper) constructor.newInstance(new Object[] {}));
+      setHelper((XmlValidationHelper) constructor.newInstance(new Object[] {}));
```

Comment:
This code isn't correct: it's resolving the className from the calling bundle's context, rather than the bundle contributing the extension.  It should use `config.createExecutableElement(className)`.

Reply:
File a new bug? I think we need to fix this in a separate PR. There may be more instances like this.

Reply:
#2874